docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: "3"
+
+services:
+  code-server:
+    container_name: code-server
+    image: codercom/code-server
+    ports:
+      - "8443:8443"
+    volumes:
+      - "${PWD}:/home/coder/project"
+    environment:
+      PASSWORD: ${PASSWORD}
+    command: --allow-http --no-auth
```

Comment:
You have `PASSWORD` env var, but it will not work, because the `--no-auth` flag is set.

Reply:
Should add a comment here indicating that you need to remove --no-auth with a docker-compose override file or edit this one if you want to use $PASSWORD.